web_service/xmlrpc_test.py

```diff
@@ -0,0 +1,32 @@
+import xmlrpc.client
+
+server = "localhost"
+port = 8069
+db_name = "practice"
+user = "admin"
+password = "admin"
```

Comment:
Use uppercase for constants please.

```suggestion
SERVER = "localhost"
PORT = 8069
DB_NAME = "practice"
USER = "admin"
PASSWORD = "admin"
```